rasa/nlu/selectors/response_selector.py

```diff
@@ -262,9 +267,9 @@ def _create_retrieval_intent_mapping(
 
         retrieval_intent_mapping = {}
         for example in training_data.intent_examples:
-            retrieval_intent_mapping[
-                example.get(RESPONSE)
-            ] = f"{example.get(INTENT)}/{example.get(RESPONSE_KEY_ATTRIBUTE)}"
+            retrieval_intent_mapping[example.get(RESPONSE)] = example.get(
```

Comment:
please add a function description of what the dict is that get's returned here

Reply:
Removed the usage of this function.